package.json

```diff
@@ -41,9 +44,11 @@
     "eslint-plugin-sonarjs": "^0.23.0",
     "husky": "^9.0.7",
     "lint-staged": "^15.2.1",
+    "postcss": "^8.4.33",
     "prettier": "3.2.4",
     "prettier-plugin-prisma": "^5.0.0",
     "prisma": "^5.8.1",
-    "sass": "^1.70.0"
+    "sass": "^1.70.0",
```

Comment:
Can we delete sass completely?

Reply:
Just double checked if everything was going to work without that sass dependency. It worked! So I think it can be removed from package.json